ludwig/data/preprocessing.py

```diff
@@ -981,7 +998,8 @@ def preprocess_for_training(
             skip_save_processed_input=False,
             preprocessing_params=default_preprocessing_parameters,
             backend=LOCAL_BACKEND,
-            random_seed=default_random_seed
+            random_seed=default_random_seed,
+            callbacks=[]
```

Comment:
Let's replace `[]` with `None` to avoid mutable default issue.

Reply:
Done.